addons/web_editor/static/src/js/editor/odoo-editor/test/spec/insert.test.js

```diff
@@ -90,6 +93,21 @@ describe('insert HTML', () => {
                 contentAfter: '<pre>abcdef[]<br>ghi</pre>',
             });
         });
+        it('should unpack elements from a lone paragraph', async () => {
+            // TODO remove this test if/when the execCommand ends up not
+            // unpacking content of a paragraph or from the first and last
+            // inserted elements (in case there are more than one). Don't forget
+            // to update the appendBehaviorBlueprint method in Knowledge
+            // wysiwyg.js at that point.
```

Comment:
It would probably be better to fix this rather than crystalize the wrong behavior inside a test. We could move the two conditions that test `container.childElementCount` from `insert` into `_onPaste`. I had a quick look and I don't think commands other than insert _itself_ rely on this behavior. The only dubious case I found that needs extra checks is the `/sign` command.